src/overlays/actors/ovl_Bg_Ice_Shutter/z_bg_ice_shutter.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_bg_ice_shutter.h"
+#include "sfx.h"
```

Comment:
no need to include "sfx.h" I just mentioned it because it's where the enum was, it's already included (through `z64.h` from `variables.h` from `global.h` from `z_bg_ice_shutter.h`)